dnf5.spec

```diff
@@ -240,11 +248,15 @@ License:        LGPL-2.1-or-later
 Requires:       libsolv%{?_isa} >= %{libsolv_version}
 Requires:       librepo%{?_isa} >= %{librepo_version}
 Requires:       sqlite-libs%{?_isa} >= %{sqlite_version}
+Conflicts:      dnf-data < 4.16.0
```

Comment:
DNF5 should conflict with any version of `dnf-data` before ownership of /etc/dnf/dnf.conf was moved to libdnf5.